README.md

```diff
@@ -54,11 +54,12 @@ core [source code master repository](https://github.com/nordic-institute/X-Road/
 
 The tools listed below are used in the development of the X-Road.
 
-| URL | Description                                                                |
-| --- |----------------------------------------------------------------------------|
-| https://nordic-institute.atlassian.net/browse/XRDDEV | X-Road backlog.                                                            |
-| https://nordic-institute.atlassian.net/wiki/spaces/XRDKB | X-Road Knowledge Base.                                                     |
-| https://nordic-institute.atlassian.net/servicedesk/customer/portal/4 | X-Road Service Desk for submitting enhancement requests and error reports. |
+| URL                                                                     | Description                                                                |
```

Comment:
Let's add the X-Road Service Desk for reporting security issues. Also, this could be updated:

> [Sign up](https://id.atlassian.com/signup) for an account and
get access to the tools.

=>

> [Sign up](https://id.atlassian.com/signup) for an account and
get access to the X-Road Service Desk.